src/components/molecules/FabIcon.vue

```diff
@@ -63,6 +63,11 @@
 				<BaseIcon :icon="action.icon" :source="action['icon-source']" />
 			</base-button>
 		</div>
+		<div
+			:class="{
+				overlay: isOpen,
+			}"
+		></div>
```

Comment:
This should work but can be even cooler if you use a vue transition. Then the overlay could easaly get faded in.
https://vuejs.org/v2/guide/transitions.html

It would look something like:
```suggestion
		<transition  name="fade">
			<div v-if="isOpen" class="overlay" />
		</transition>
```
and in the styles section
```css
.fade-enter-active, .fade-leave-active {
  transition: opacity var(--transition-medium);
}
.fade-enter, .fade-leave-to {
  opacity: 0;
}
```

Reply:
Cool! Makes sense. Thanks for the tip.